src/pybind/ceph_argparse.py

```diff
@@ -23,6 +23,9 @@
 import uuid
 
 
+FLAG_MGR = 8   # command is intended for mgr
```

Comment:
already some flags in ceph.in; we should find a better place if they need sharing